presto-docs/src/main/sphinx/release/release-0.101.rst

```diff
@@ -21,3 +21,7 @@ Hive Changes
 * Lower the Hive metadata refresh interval from two minutes to one second.
 * Invalidate Hive metadata cache for failed operations.
 * Support ``s3a`` file system scheme.
+* Make Parquet file column access configurable. By default, Parquet file column is accessed via column index.
```

Comment:
"By default, Parquet file columns are accessed by their ordinal position in the Hive table definition."